app/src/main/java/com/teamagam/gimelgimel/app/network/services/GGMessagingUtils.java

```diff
@@ -1,5 +1,6 @@
 package com.teamagam.gimelgimel.app.network.services;
 
+import android.content.Context;
 import android.util.Log;
 
 import com.teamagam.gimelgimel.app.model.ViewsModels.Message;
```

Comment:
I;m having a hard time deciding myself, so I'm sharing my thoughts:

This class seems like it should be refactored, which will effect other parts of the codebase.

The problem right now is that all of these utility classes share the same structure, and it can be resolved by refactoring it to a GGMessageSender class (or whatever name you see fit) that will expose all the needed functionality (different type of messages sending).

It wont be a huge change, but I think it would clean up the code a bit, and stop this (soon to be) monstrous class evolutions